src/viewer.js

```diff
@@ -1944,7 +1944,28 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         this.raiseEvent( 'clear-overlay', {} );
         return this;
     },
+    
+     /**
+     * Finds an overlay identified by the reference element or element id
+     * and returns it as an object, return null if not found.
+     * @method
+     * @param {Element|String} element - A reference to the element or an
+     *      element id which represent the ovelay content to be removed.
```

Comment:
Does this method only get used when you are going to remove an overlay? The comment seems to suggest it does but I can think of other reasons it could be useful.